### PR TITLE
Simplified provisioning setup

### PR DESCRIPTION
**Pull request by tosan88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,20 +13,57 @@ Set up SSH
 
 See [SSH_README.md](/SSH_README.md/)
 
+Provision a delivery cluster
+------------------------------
+
+```bash
+## Set all the environment variables required to provision a cluster. These variables are stored in LastPass
+## For PROD cluster
+## LastPass: Delivery cluster provisioning setup
```
Comment (tamas-molnar) [inline README.md]: Can you prefix this note with the env name too please?

File: README.md [hunk 2]
```diff
@@ -13,20 +13,57 @@ Set up SSH
...
+    -e "CLUSTER_BASIC_HTTP_CREDENTIALS=$CLUSTER_BASIC_HTTP_CREDENTIALS" \
+    coco/coco-provisioner:v1.0.0
+
+## If the cluster is running, set up HTTPS support (see below)
+```
+
+Getting started
+---------------
 
 Building
```
Comment (tamas-molnar) [inline README.md]: The building section isn't necessary anymore, for a provisioning there's no need to checkout this project anymore.

File: README.md [hunk 3]
```diff
@@ -92,7 +129,7 @@ docker run \
     -e "BRIDGING_MESSAGE_QUEUE_PROXY=$BRIDGING_MESSAGE_QUEUE_PROXY" \
     -e "API_HOST=$API_HOST" \
     -e "CLUSTER_BASIC_HTTP_CREDENTIALS=$CLUSTER_BASIC_HTTP_CREDENTIALS" \
-    coco-provisioner
+    coco/coco-provisioner
```
Comment (tamas-molnar) [inline README.md]: Version?

File: README.md [hunk 4]
```diff
@@ -13,20 +13,57 @@ Set up SSH
```
Comment (tamas-molnar) [inline README.md]: Can you also adjust the tutorial please? It still references checking out this project.

Comment (tosan88) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (tosan88) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (tosan88) [inline README.md]: As discussed, I created a separate readme for devs

File: README.md (hunk 3 above)
Comment (tosan88) [inline README.md]: As discussed, I created a separate readme for devs

**Pull request merged**